File: problems/problem04/problem04.cpp
#include "utility.h"

int two_sum(int* a, int sz, int target) {
    /* fill me in */
    return 0;
}

int main() {
    int sz;
    int* a = read_ints(stdin, &sz);
    printf("%d\n", two_sum(a, sz - 1, a[sz - 1]));
}

File: problems/problem04/problem04.js
function two_sum(a, target) {
    /* fill me in */
}


function main() {
    let fs = require('fs');
    let stdin = fs.readFileSync(0).toString();
    let lines = stdin.trim().split('\n');
    let numbers = lines.map(x => parseInt(x));
    let search_value = numbers.pop();
    let result = two_sum(numbers, search_value);
    console.log(result);
}

main();
